WebHostLib/templates/multitracker__SMW.html

```diff
@@ -0,0 +1,144 @@
+{% extends "multitracker.html" %}
+{% block head %}
+    {{ super() }}
+    <script type="application/ecmascript" src="{{ url_for("static", filename="assets/jquery.scrollsync.js") }}"></script>
+    <script type="application/ecmascript" src="{{ url_for("static", filename="assets/lttpMultiTracker.js") }}"></script>
+{% endblock %}
+
+{# List all tracker-relevant icons. Format: (Name, Image URL) #}
+{% set icons = {
+    "Big Mario"                         : "https://github.com/PoryGone/SMW_AP_Tracker/blob/main/super_mario_world_randomizer_porygone/images/items/big_mario.png?raw=true",
```

Comment:
If the image is changed or removed from Pory's GitHub, the tracker layout may break. Consider alternative hosting methods.

Reply:
Put into my ap-icons repository (https://github.com/mrkssr/ap-icons) which is already used for the Undertale tracker. Pory is credited in tracker code and also mentioned in a text file in the repo.